test/unit/transforms/test_summarization_accuracy_metrics.py

```diff
@@ -128,6 +128,31 @@ def test_bert_score_call_with_bertscore_model_object():
     mock_bertscore_model.get_helper_scores.assert_called_once_with("Hello there!", "Hi")
 
 
+def test_bert_score_call_with_target_output_keys_provider():
```

Comment:
Please add a unit test where there are multiple target outputs, and verify that the max score is returned.